src/compiler/interface.jl

```diff
@@ -80,6 +80,8 @@ end
 
 Base.copy(ps::Params) = union!(Params(), ps)
 Base.union(ps::Params, itrs...) = union!(copy(ps), itrs...)
+Base.issetequal(ps1::Params, ps2::Params) = issetequal(ps1.params, ps2.params)
+Base.issetequal(ps1::Params, x::Base.AbstractSet) = issetequal(ps1.params, x)
```

Comment:
Should we also define `issetequal(x::Base.AbstractSet, ps1::Params)` for generality (even though internally we don't use it so far)?

Reply:
```suggestion
Base.issetequal(x::Base.AbstractSet, ps1::Params) = issetequal(x, ps1.params)
```